pyproject.toml

```diff
@@ -109,11 +123,18 @@ ban-relative-imports = "all"
 "onnx/reference/ops/**" = ["N801"] # Class casing
 "onnx/reference/ops/_op_list.py" = ["F401"]
 "onnx/__init__.py" = ["F401"]
+"onnx/npx/__init__.py" = ["F401", "F403"]
+"onnx/npx/npx_functions.py" = ["F821"]
+"onnx/npx/npx_functions_test.py" = ["F821"]
+"onnx/npx/npx_graph_builder.py" = ["N811", "N801"]
```

Comment:
Possible to add a comment on why they need to be ignored? I see codes that aren’t in the list. If possible I would just noqa in-line so they are not ignored for the whole file. I also hope to see we reduce this ignore list instead of growing it.

Reply:
mypy does not work well with dynamic types (see [_make_type](https://github.com/onnx/onnx/pull/4975/files#diff-4172614a6538ec08dffe5b2799444c67fb64835cfbac8e3d82082f4472b838baR566). lintrunner gives different results on the machine I use comare to CI. I'll do better if the PR reaches a consensus.